### DISCUSSION TITLE
Recreating a sky view in 1943, effect of Atmosphere Visualisation option

### DISCUSSION THREAD

**Discussion by Pennine-Lad** [Stellarium/stellarium#3417] [General]
Hi, for an historical study I'm trying to visualise the sky an observer would have seen at the time and location below. I'm a novice with Stellarium and not understanding whether the Atmosphere Visualisation option should be set for the most accurate view as this completely changes what could be seen.

Gibraltar, standing on the shore looking East over the sea. N 36.14 deg W 5.35 deg
July 7th 1943 2307 wartime double summer time so 2107 UTC.

An astronomical almanac suggests nautical twilight ended 18 mins earlier at 2049 UTC see image at dir link (Gib-1). So quite dark. 
But if switch on Atmosphere Visualization with for a dark sky setting on slider it is quite light (Gib-2) and does not become dark until some hours later. See screenshots. Gib-3 without Visualization on, mag 6 stars.

[https://tinyurl.com/2us39nw3]

Thanks for any insight!
David

**Comment by gzotti** [comment]
Repeat Gib-2.jpg with your Location panel displayed, please.

**Reply by Pennine-Lad** [reply]
Hi, ah your question made me realise the novice mistake that I made. Clicked Gibraltar, N. Europe which is a town in the UK. Gibraltar, Spain gives a dark sky with Atmosphere Visualisation added and class 2 dark sky.

Thanks for your help.

David

![Gib-2-correct-location-atmos-vis-on](https://github.com/Stellarium/stellarium/assets/145571058/16b64fb5-38a2-477e-9a79-b37a66c9ef9f)
